app/Http/Controllers/Api/SettingsController.php

```diff
@@ -28,120 +28,9 @@ class SettingsController extends Controller
 {
 
 
-    public function ldaptest()
```

Comment:
Can we also kill `postLdapSettings()` in `SettingsController` (along with the route) since we're handling it in the Livewire component now?